src/tools/tidy/src/run_make_tests.rs

```diff
@@ -0,0 +1,98 @@
+//! Tidy check to ensure that no new Makefiles are added under `tests/run-make/`.
+
+use std::collections::BTreeSet;
+use std::fs::File;
+use std::io::Write;
+use std::path::{Path, PathBuf};
+
+pub fn check(root_path: &Path, bless: bool, bad: &mut bool) {
+    let tests_path = &root_path.join("tests");
+
+    let allowed_makefiles = {
+        let allowed_makefiles = include!("expected_run_make_makefiles.txt");
+        let is_sorted = allowed_makefiles.windows(2).all(|w| w[0] < w[1]);
```

Comment:
i think you messed up something here, you arent actually parsing the file
can you just reuse the code from the issues test stuff?

Reply:
oh, `include!` does rust syntax, please leave a comment about that lol

Reply:
Added a comment explaining why `include!` is used.